lib/car.rb

```diff
@@ -0,0 +1,31 @@
+class Car
+  attr_reader :name, :monthly_payment, :loan_length
+
+  def initialize(name, monthly_payment, loan_length)
+    @name = name
+    @monthly_payment = monthly_payment
+    @loan_length = loan_length
+  end
+
+  def make
+    @name = @name.split(" ")
+    return @name[0]
+  end
+
+  def model
+    @name = @name.split(" ")
+    return @name[1]
+  end
```

Comment:
The problem with these are that you are overwriting the instance variable @name when `make` or `model` are called. It would make more sense to use local variables that scoped only to these methods:

```
def make
  name = @name.split(" ")
  return name[0]
end
```

or even using a different variable name if that is more clear

```
def make
  make_model = @name.split(" ")
  return make_model[0]
end
```